config2.json

```diff
@@ -34,6 +34,35 @@
       "codesearch_port": 8082
     },
 
+    "mozilla-esr115": {
+      "priority": 20,
+      "on_error": "continue",
+      "cache": "codesearch",
+      "index_path": "$WORKING/mozilla-esr115",
+      "files_path": "$WORKING/mozilla-esr115/git",
+      "git_path": "$WORKING/mozilla-esr115/git",
+      "git_blame_path": "$WORKING/mozilla-esr115/blame",
+      "github_repo": "https://github.com/mozilla/gecko-dev",
+      "hg_root": "https://hg.mozilla.org/releases/mozilla-esr115",
+      "objdir_path": "$WORKING/mozilla-esr115/objdir",
+      "codesearch_path": "$WORKING/mozilla-esr115/livegrep.idx",
+      "codesearch_port": 8083
```

Comment:
should this use a different port than esr102?

Reply:
Yes!

Reply:
Fixed this